packages/ui-billboard/babel.config.js

```diff
@@ -23,11 +23,6 @@
  */
 
 module.exports = {
-  shouldPrintComment(value) {
-    const isTsRelatedComment = /@ts-expect-error|FIXME/.test(value)
-
-    return !isTsRelatedComment
-  },
```

Comment:
There are a few ts-expect-errors left in the `Billboard.examples.tsx` :) Please resolve those as well! When typing, we have to check the test and examples files as well.

Reply:
Please resolve the `ts-expect-error` comments in the `Billboard.examples.tsx` too!